### PR TITLE
Troubleshooting and some improved logging

### PR DESCRIPTION
**Pull request by davenewza**
Improved docs and added some nicer output logging.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# docker build -t pulumi-go:alpha .
```
Comment (peetjvv) [inline Dockerfile:1]: would you mind adding an example docker run command here as well please?

**Review by peetjvv — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -25,12 +25,22 @@ docker run --rm -it `
...
+
+  1. `az login` to authenticate the Azure CLI with your user account.
+  2. `az acr login -n whereismytransport` to authenticate with our ACR.
+  3. `docker pull whereismytransport.azurecr.io/pulumi-go:latest` to pull the latest pulumi-go image.
+
 ## Future work
 
 This is very alpha. Some future work to consider:
 
+ - Reduce the size of the image.  It's currently sitting at 1.9GB.
```
Comment (peetjvv) [inline README.md:44]: this can be quite a hindrance if you don't have good internet speeds

File: Dockerfile (hunk 1 above)
Comment (davenewza) [reply Dockerfile]: Added.  There is a lot more detail in the README since it's not that trivial.

File: README.md (hunk 2 above)
Comment (davenewza) [reply README.md:44]: Yeah, although I don't really see us running this locally much at all.  Let's see how it goes.

**Pull request merged**